src/main/java/service/PriceCatalogHorizontal.java

```diff
@@ -3,4 +3,6 @@
 public interface PriceCatalogHorizontal {
 
     Double getColorPrice(int color);
+
+    double getDollarExchangeRate();
```

Comment:
Please, fix the same implementation in two classes "PriceCatalogHorizontal" and "PriceCatalogVertical"

Reply:
done